db-scripts/src/main/resources/cgds.sql

```diff
@@ -694,7 +694,7 @@ CREATE TABLE `sample_cna_event` (
 
 -- --------------------------------------------------------
 CREATE TABLE `copy_number_seg` (
-  `SEG_ID` int(255) NOT NULL auto_increment,
+  `SEG_ID` BIGINT(20) NOT NULL auto_increment,
```

Comment:
other tables use "bigint(20)" rather than BIGINT(20) ... so probably better to stay consistent with casing of types